APMrover2/Log.cpp

```diff
@@ -284,7 +284,7 @@ void Rover::Log_Write_RC(void)
     logger.Write_RCIN();
     logger.Write_RCOUT();
     if (rssi.enabled()) {
```

Comment:
It may be worth moving this call to live inside of logger, as it reduces the per vehicle boilerplate.